application/src/jvmMain/kotlin/replace/usecase/temporaryfileupload/CreateTemporaryFileUploadUseCase.kt

```diff
@@ -0,0 +1,39 @@
+package replace.usecase.temporaryfileupload
+
+import replace.datastore.FileStorage
+import replace.datastore.TemporaryFileRepository
+import replace.dto.TemporaryFileUploadDto
+import replace.dto.toDto
+import replace.model.TemporaryFile
+import java.io.InputStream
+import java.net.URLConnection
+
+object CreateTemporaryFileUploadUseCase {
+
+    suspend fun execute(
+        fileName: String,
+        input: InputStream,
+        temporaryFileRepository: TemporaryFileRepository,
+        fileStorage: FileStorage
```

Comment:
Trailing comma

Reply:
Fixed manually, but such styling errors should be fixed automatically by ktlint when possible.

Reply:
https://github.com/pinterest/ktlint/issues/709

Reply:
I would add the rule directly on the master later or another branch, not in this one